src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorMapToDocumentEditsResponse.cs

```diff
@@ -9,6 +9,6 @@ internal class RazorMapToDocumentEditsResponse
     {
         public TextEdit[] TextEdits { get; set; }
 
-        public long HostDocumentVersion { get; set; }
+        public int? HostDocumentVersion { get; set; }
```

Comment:
Are these necessary changes? Or cleanups for consistency?

Reply:
Should be evaluating which pieces make sense to be nullable and which pieces don't. I know O# went from `long` -> `int?` but we should be very studious on where we specify nullable or not because it indicates what an API is capable of (our usage can differ from O#s)